packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/semanticdata/listeners/SemanticDataInitializer.java

```diff
@@ -38,6 +38,6 @@ public SemanticDataInitializer(ISemanticDataCreationService semanticDataCreation
     @Transactional
     @TransactionalEventListener
```

Comment:
`@TransactionalEventListener(propagation = Propagation.REQUIRES_NEW)` required so that other listeners can react to the `SemanticDataCreatedEvent` send during this.